RLTest/env.py

```diff
@@ -258,7 +260,10 @@ def assertNotEqual(self, first, second, depth=0):
         self._assertion('%s != %s' % (first, second), first != second, depth)
 
     def assertOk(self, val, depth=0):
-        self.assertEqual(val, 'OK', depth + 1)
+        if isinstance(val, (bytes, bytearray)):
+            self.assertEqual(val, b'OK', depth + 1)
+        else:
+            self.assertEqual(val, 'OK', depth + 1)
```

Comment:
I do not think this is how we should handle the binary string issue because this only solve the assertOk case. we should handle this in the _assertion method with a more generic approach.

Reply:
I have used a more generic approach - it is possible to set decoding at Env construction time.

assertOk is the *only* method here where we *know* the checked input is very likely to be string, since we are comparing to the string 'Ok' ... in all the other cases, it is unclear.  So as an additional  convenience above-and-beyond the decode_responses mechanism provided by the redis module, I've added this too.  It could be removed in favour of complete reliance on decode_responses, but I don't think that's ideal.